### ISSUE TITLE
import docker-machine-kvm

### ISSUE THREAD

**Issue by SvenDowideit** [machine-drivers/discussion#2]
from https://github.com/dhiltgen/docker-machine-kvm/issues/67

I'd suggest renaming it to something that contains `libvirt` - perhaps `machine-driver-libvirt`

this project was the trigger to creating this org, so I'm presuming its a simple yes :)

**Comment by r2d4** [comment]
I'd like to see some of the code replaced or merged with https://github.com/kubernetes/minikube/tree/master/pkg/drivers/kvm

I don't exactly remember what the differences were, but in minikube we've actively maintained this one and tested it for the last year.

**Comment by afbjorklund** [comment]
At least `libvirt` is a better name than `kvm2` ?

**Comment by r2d4** [comment]
Yeah absolutely - the name should be `libvirt` to correspond with the new `qemu` driver.  `kvm` and `kvm2` should be dropped.

**Comment by dhiltgen** [comment]
I'm :+1: on transitioning the code here and renaming it.

**Comment by brgnepal** [comment]
Cool @dhiltgen Thanks

**Comment by praveenkumar** [comment]
@dhiltgen @r2d4 @SvenDowideit Since everyone agreed can we migrate the driver repo now to this location?

**Comment by zakame** [comment]
Please, let's migrate!

**Comment by gbraad** [comment]
I would also like to see this happen. @dhiltgen, any concerns or remaining topics to agree on?

**Comment by dhiltgen** [comment]
> I would also like to see this happen. @dhiltgen, any concerns or remaining topics to agree on?

I don't have any concerns.  Once the code is live on the new repo and there's a release, could someone file a pr against my repo to update the top-level readme to point to the new tree, releases, etc. so we can steer people to the new home?

**Comment by gbraad** [comment]
I believe if you do a transfer, the issue tracker remains active (and forks
remain identifiable). If a new repo is created, we have a fork visible from
the github interface.


-- 

   Gerard Braad | http://gbraad.nl
   [ Doing Open Source Matters ]

**Comment by gbraad** [comment]
If you do a transfer, you can create a new repo and update this with a
readme.md without having to wait for us ;-)




-- 

   Gerard Braad | http://gbraad.nl
   [ Doing Open Source Matters ]

**Comment by SvenDowideit** [comment]
@dhiltgen as you're an owner of both the repo, and this org, do you want to just transfer ownership of this repo into the machine-driver org?

If you make me an admin of this repo, I can do it for you too

as @gbraad says, that way we also xfer the issues and prs :)

**Comment by praveenkumar** [comment]
Hey @SvenDowideit , Does @dhiltgen provided you the ownership of the driver repo? Can we put it under this organization soon since we also have to put another PR which will make sure kvm2 driver enhancements are also part of it?

**Comment by afbjorklund** [comment]
Guess it got imported. Lost the issues and got new naming though, that was a bit unfortunate.

**Comment by gbraad** [comment]
It is a fork... :-(

-- 

   Gerard Braad | http://gbraad.nl
   [ Doing Open Source Matters ]

**Comment by SvenDowideit** [comment]
yup - @dhiltgen didn't reply, so I figured we're better off with some momentum
if he comes back later, we can rename the fork, then move the repo etc...

**Comment by gbraad** [comment]
Exactly, same thought here. Was considering to do the same. Anyway, a
transfer can always happen under a different name and a force push...
However, issues recorded might be duplicated. Moving forward is more
important at this moment.

What are the most urgent concerns to address? Let's create an issue for
this and leave this open for a little while longer in case @dhiltgen
reaches out again.

-- 

   Gerard Braad | http://gbraad.nl
   [ Doing Open Source Matters ]

**Comment by tunggad** [comment]
Hi,
is there any action in progress according to this discussion?

We are approaching end of 2018 and i still have to use @dhiltgen docker-machine-kvm at version 0.10.0 required by minishift (1.27.0) and get this error when trying to start minishift or create vm with docker-machine create -d kvm:

```
Failed to decode dnsmasq lease status: unexpected end of JSON input
```

My system is an arch linux box with following setup:
```
[cryto@archserver ~]$ virsh version
Compiled against library: libvirt 4.8.0
Using library: libvirt 4.8.0
Using API: QEMU 4.8.0
Running hypervisor: QEMU 3.0.0

[cryto@archserver ~]$ dnsmasq --version
Dnsmasq version 2.80  Copyright (c) 2000-2018 Simon Kelley
Compile time options: IPv6 GNU-getopt DBus i18n IDN2 DHCP DHCPv6 no-Lua TFTP conntrack ipset auth DNSSEC loop-detect inotify dumpfile

This software comes with ABSOLUTELY NO WARRANTY.
Dnsmasq is free software, and you are welcome to redistribute it
under the terms of the GNU General Public License, version 2 or 3.

[cryto@archserver ~]$ docker-machine --version
docker-machine version 0.16.0, build HEAD

[cryto@archserver ~]$ virsh --connect qemu:///system net-list
 Name              State    Autostart   Persistent
----------------------------------------------------
 default           active   yes         yes
 docker-machines   active   yes         yes
```

I'm very happy if someone can help me get minishift starting properly

**Comment by gbraad** [comment]
You are right that this is taking time... but this is now out of our
control.

Note: The error you get will not be automatically solved when we would have
solved the transfer (and released a new build), as the issue you rae having
is related to malformed JSON. please report the issue with minishift to
discuss further. Regarding the comment as off-topic.

**Comment by tunggad** [comment]
I found this issue in Fedora Bugzilla reported back in June 2015: https://bugzilla.redhat.com/show_bug.cgi?id=1228631

It states that libvirt for some reasons from particular version does not use the dnsmasq managed lease database (the .leases files) anymore, that's why for the libvirt-managed networks there is also no .leases file -> no file under /var/lib/libvirt/dnsmasq/docker-machines.leases for docker-machines network -> kvm driver gets the above error

So i wonder, how could the kvm driver of @dhiltgen work for minishift since then?
